examples/kube/centralized-logging/efk/run.sh

```diff
@@ -18,7 +18,6 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 ${DIR}/cleanup.sh
 
-${CCP_CLI?} create -f ${DIR?}/elasticsearch-statefulset.yaml
```

Comment:
This Elasticsearch StatefulSet needs to stay.

Reply:
pushed run.sh with elasticsearch-statefulset.yaml put back.  Thanks for catching that.